.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
 
 env:
   - TEST_RUN="ansible-playbook -i herp, bastion.yml install-ci.yml provision.yml tests/validate-ci.yml --syntax-check"
-  - TEST_RUN="ansible-playbook -i inventory/allinone bastion.yml install-ci.yml tests/validate-ci.yml -c docker -vv -e @secrets.yml.example --skip-tags monitoring,not-on-docker,letsencrypt"
+  - TEST_RUN="ansible-playbook -i inventory/allinone bastion.yml -c docker -vv -e @secrets.yml.example --skip-tags monitoring,not-on-docker,letsencrypt"
```

Comment:
Since we had two test runs that both ran `install-ci.yml` (lines 15 and 16), but only one of the two also runs `bastion.yml`, I modified the first to only run `bastion.yml`, leaving the second to only run `install-ci.yml`.